frontends/PyCDE/src/pycde/testing.py

```diff
@@ -0,0 +1,43 @@
+from pycde import System, module
+
+import builtins
+import inspect
+
+
+def testmodule(generate=True,
+               print=True,
```

Comment:
Is this intended just for internal testing or is it also intended for users? If the latter, `print` should default to false, and emit_outputs should default to true. If the former, it should be moved to the test folder.

Reply:
I don't really see a use case for users.

Reply:
+1 on no use-case for users. I'll move it to the test folder.